web/cypress/e2e/TestRunDetail/Outputs.spec.ts

```diff
@@ -94,7 +93,7 @@ describe('Outputs', () => {
     cy.get('[data-cy=output-pending-tag]').should('have.length', 1);
 
     // Revert
-    cy.get('[data-cy=output-reset-button]').click();
+    cy.get('[data-cy=trace-actions-revert-all]').click();
```

Comment:
I don't know if here is a good place for it, but question: by applying the "Revert All" action, it makes sense to query the user before doing it (something like "Are you sure that you want to revert the last X changes?")?
It seems that this is the type of action that if the user clicks by mistake they can undo a good bunch of their work.